docs/general/status.rst

```diff
@@ -40,6 +47,44 @@ consider waiting until it is more mature.
    Workstation that may hinder your ability to use it at any functional
    level.
 
+SecureDrop Workstation Feature Comparison
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+SecureDrop Workstation provides a faster and more streamlined way of interacting
+with sources and submissions, while preserving important security properties.
+That said, it is still under development, and not all features available on the
+Tails-based (existing) system are available in SecuredDrop Workstation. We are
+working towards feature parity, but in the meantime, there are some notable differences.
+
+The following features have not yet been implemented:
+
+* Viewing download progress/download speed
+* Cancelling downloads
+* Network speed indicators
+* Searching/filtering of sources by codename
+* User-selection of application in which to view submissions (submissions
+  are viewed in one default application depending on their filetype)
```

Comment:
```suggestion
  are viewed in a pre-determined application depending on their filetype)
```
I feel that "one default application" is a bit confusing here. Perhaps using "pre-determined application" might be clearer?